test/jest/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I think this can be avoided by using 
```
"devDependencies": {
   "react-router5": "npm:react-router-dom@^5.x.y",
   "react-router6": "npm:react-router-dom@^6.x.y"
}
```

ref: https://stackoverflow.com/a/56495651

Reply:
Thanks! This works